foursq.js

```diff
@@ -0,0 +1,51 @@
+class Place {
+  constructor(name, cat, lat, lng, url, hereNow) {
+    this.name = name;
+    this.cat = cat;
+    this.lat = lat;
+    this.lng = lng;
+    this.url = url;
+    this.hereNow = hereNow;
+  }
+}
+
+// FourSquare API
+function fourSquareFunction(search) {
+  var places = [];
+
+  var radius = 8000; // meters
+
+  // on checkbox click trigger this search!!!
+
+  var url = 'https://api.foursquare.com/v2/venues/search?' +
+    'll=38.894470,-77.036583&' +
+    'client_id=PDKD4SZGV2WUM3HW00FFLWJGUMMLSFLMG4UGKN4DEGDH0GWB&' +
+    'client_secret=JWQ5SO32GUFAKVDX5QWE1PPIHSKER40VTOWT01PUSA1O42TS&' +
+    'v=20170101&' + // time to be changed...
```

Comment:
When you merge, you will want to update this to use the moment js that is in the develop branch already

Reply:
Right, I saw what you did in yours. Very good! I will do that in the merge